### ISSUE TITLE
Library doesn't support plain react-native for iOS

### ISSUE THREAD

**Issue by markiian-rolique** [nandorojo/burnt#24]
This library doesn't support plain react-native, as root dependencies relies on expo module system, so it's impossible to use it with the plain react-native setup. Would be great if there will be possibility to install it to react-native project without expo

**Comment by firmnardians** [comment]
That's right, this package can only be used for React Native which uses Expo.

**Comment by nandorojo** [comment]
it just depends how you define this. if you’re willing to `yarn add expo-modules-core` and do a few additional steps it will work with react native CLI.

the react native community cli is an alternate approach with its own trade offs. expo modules makes life way easier for library authors, and they support both new and old architectures, as well as fabric, with very little config for authors. competing starters simply do not have the same DX for library maintainers. this is why almost all RN libraries with native code go stale and stop getting maintenance.

so if you’re willing to install expo modules into your RN CLI app, which is fully doable, it will indeed work.
